app/questionnaire/placeholder_transforms.py

```diff
@@ -59,15 +59,14 @@ def format_date(self, date_to_format: str, date_format: str) -> str:
 
     @staticmethod
     def format_list(list_to_format: Sequence[str]) -> str:
-        formatted_list = "<ul>"
-        for item in list_to_format:
-            formatted_list += f"<li>{item}</li>"
+        formatted_list = "<ul>" + "".join(
+            [f"<li>{item}</li>" for item in list_to_format]
```

Comment:
This list comp isn't needed, can just be:
```py
"".join(f"<li>{item}</li>" for item in list_to_format)
```

Reply:
Would we not need the `<ul> </ul>` anymore?

Reply:
Changed 👍